From 03f58d5c51c9dc68c60baa917e750f697814fb73 Mon Sep 17 00:00:00 2001
From: Sebastian Harl <sh@tokkee.org>
Date: Fri, 21 Nov 2008 16:33:39 +0100
Subject: [PATCH] conf/NFSROOT: Added git-core.

This is required when accessing the FAI config space through Git.
---
 conf/NFSROOT |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/conf/NFSROOT b/conf/NFSROOT
index 768c68d..057ad90 100644
--- a/conf/NFSROOT
+++ b/conf/NFSROOT
@@ -7,7 +7,7 @@ hwinfo psmisc pciutils hdparm smartmontools parted mdadm lvm2
 dnsutils ntpdate dosfstools cvs jove xfsprogs xfsdump
 procinfo dialog discover mdetect
 console-tools console-common
-expect iproute udev subversion
+expect iproute udev subversion git-core
 
 # needed for the simple examples
 cfengine2 libapt-pkg-perl
-- 
1.6.0.3.523.g304d0

